channels/stable

```diff
@@ -133,6 +136,10 @@ spec:
     recommendedVersion: 1.11.10
     requiredVersion: 1.11.10
   kopsVersions:
+  - range: ">=1.25.0-alpha.1"
+    recommendedVersion: "1.25.0"
```

Comment:
Should this be changed all the way back to K8s 1.20?

Reply:
No, we do that once .1 is released.

Reply:
Ah. TIL ...